_posts/2019-09-07-best-practice-ansible-configuration-for-kolla.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Best Practice Ansible Configuration for Kolla"
+date: 2019-09-07 18:15:15
+image: '/assets/img/'
+type: post
+published: true
+status: publish
+tags:
+- kolla-ansible
+- ansible
+- openstack
+categories:
+- Chia sẻ kinh nghiệm
+author:
+  login: Daikk115
+  email: daikk115@vietopeninfra.org
+  display_name: Daikk115
+  first_name: Dai
+  last_name: Dang Van
+---
+
+Chào các bác, lại là tôi @daikk115 đây!
+
+**Tóm tắt**
```

Comment:
theo [markdownlint rule](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md) thì không để bold text làm section header nhé

Reply:
Dạ, chỗ kia em cũng chỉ in đậm đoạn text thôi :D

Reply:
Thôi để em viết thành dạng khác cho thỏa mãn markdownlint :D